src/app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,15 @@
+class TagsController < ApplicationController
+  LIMIT = 30
+
+  def show
+    offset = params.fetch(:offset, 0).to_i
+
+    @tag = params[:id]
+    @functions = Tag.from_param(@tag).functions.limit(LIMIT).offset(offset).order(likes_count: :desc).all ||
+                 raise(ActiveRecord::RecordNotFound)
```

Comment:
that won't work are you expected, if the tag doesn't exist it will raise an error `unknown functions method for nil class` as `from_param` will return nil

Reply:
What do you recommend to mitigate this? Is `rescue` suitable for this?